### DISCUSSION TITLE
Scrollable List of Widgets?

### DISCUSSION THREAD

**Discussion by ThatXliner** [Textualize/textual#196] [Ideas]
It'd be cool if there was a widget like ScrollView but you give it a list of widgets as an argument.

**Comment by ThatXliner** [comment]
Mainly because I'm not sure how to implement one

**Reply by ThatXliner** [reply]
ok https://github.com/Textualize/textual/projects/1#card-66941810

**Reply by Cvaniak** [reply]
Hi! 
I was struggling with same problem so while waiting for Will's implementation you (and everyone who is looking for solution) can use [this solution](https://github.com/Cvaniak/TextualListViewUnofficial). It is hack on existing `views` but it works ( I did not have any problems with it yet but probably it can fail on some 'edge cases").

**Comment by jeffwright13** [comment]
Precisely what I had earlier been looking for. But after a quick discussion with Will, he pointed me in a different (and simpler) direction, which allowed me to get my little project off the ground. Don't be afraid to think outside the box!

**Comment by ThatXliner** [comment]
We have https://textual.textualize.io/widgets/list_view/ now! 🥳
